src/simplejavatexteditor/SupportedKeywords.java

```diff
@@ -31,7 +31,7 @@
                 "union", "while", "asm", "dynamic_cast", "namespace", "reinterpret_cast", "try",
                 "bool", "explicit", "new", "static_cast", "typeid", "catch", "false", "operator",
                 "template", "typename", "class", "friend", "private", "this", "using", "const_cast",
-                "inline", "public", "throw", "virtual", "delete", "mutable", "protected", "true", "wchar_t" };
+                "inline", "public", "throw", "virtual", "delete", "mutable", "protected", "true", "wchar_t , int_t , uint32_t , and , goto , char8_t , char16_t , char32_t" };
```

Comment:
Why would we need `uint32_t`, `int_t`, `char8_t`, `char16_t`, and `char32_t`..?

Reply:
It's C++ keywords , they are often used for finer memory management, they are not used very often, but not very rarely, so it would be better to add support for them